runtime/lua/vim/lsp.lua

```diff
@@ -938,7 +938,7 @@ end
 ---@param params table? Parameters to send to the server
 ---@param timeout_ms integer? Maximum time in milliseconds to wait for a result.
 ---                           (default: `1000`)
----@return table<integer, {err: lsp.ResponseError, result: any}>? result Map of client_id:request_result.
+---@return table<integer, {error: lsp.ResponseError, result: any}>? result Map of client_id:request_result.
```

Comment:
Technically speaking, `{ error: lsp.ResponseError?, ... }` is a correct typing here because when there is no error, the `error` field won't be set (i.e. `nil`).

Reply:
Yeah, I thought about fixing that as well but there are other functions with the same type signature that need to be changed to optional, and I would rather handle those in a PR specific to that. I will probably see about fixing those later.

Reply:
It is already fixed on master.